wallet/src/ui/app/components/receipt-card/index.tsx

```diff
@@ -74,27 +85,30 @@ function ReceiptCard({ tranferType, txDigest }: TxResponseProps) {
         </div>
     );
 
+    const statusClassName =
+        txDigest.status === 'success' ? st.success : st.failed;
+
     return (
         <>
             <div className={st.txnResponse}>
                 {txDigest.status === 'success' ? SuccessCard : failedCard}
-                <div className={cl(st.responseCard)}>
+                <div className={st.responseCard}>
                     {AssetCard}
                     {txDigest.amount && (
                         <div className={st.amount}>
-                            {txDigest.amount} <span>SUI</span>
+                            {txDigest.amount} <span>{GAS_SYMBOL}</span>
```

Comment:
Can this be any other type of coin? Is showing always SUI correct?

Reply:
Right now TransferSui is the only type that shows the amount. I think it is fine for now.